tests/keras/backend/mxnet_sparse_test.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from numpy.testing import assert_allclose
+import numpy as np
+import warnings
+import scipy.sparse as sparse
+
+from keras import backend as K
+
+
+BACKEND = None
+
+try:
+    from keras.backend import mxnet_backend as KMX
+    BACKEND = KMX
+except ImportError:
+    KMX = None
+    warnings.warn('Could not import the MXNet backend')
+
+
+class TestMXNetSparse(object):
+
+    @pytest.mark.skipif((K.backend() != 'mxnet'),
```

Comment:
You can skip all tests in a file this way - https://github.com/awslabs/keras-apache-mxnet/blob/master/tests/keras/layers/wrappers_test.py#L15
and avoid skipif for each test.

Reply:
done